modules/pipelines-document-attributes.adoc

```diff
@@ -9,4 +9,5 @@
 //
 :pipelines-title: Red Hat OpenShift Pipelines
 :pipelines-shortname: Pipelines
-:pipelines-ver: release-tech-preview-3
+:pipelines-ver: release-tech-preview-2
```

Comment:
Since this is doc for OpenShift 4.6/Pipelines 1.2, the correct value is `release-tech-preview-3`

Reply:
In that case can you tell us if this bug is valid: https://github.com/openshift/openshift-docs/issues/27742

Reply:
Also, PTAL at our Slack conversation: https://coreos.slack.com/archives/CG5GV6CJD/p1616598208300500